fixtures/app_with_framework_and_tests_and_dependencies/Tuist/Dependencies.swift

```diff
@@ -0,0 +1,8 @@
+import ProjectDescription
+
+let dependencies = Dependencies(
+    carthage: [
+        .init(name: "Alamofire/Alamofire", requirement: .exact("5.0.4"), platforms: [.macOS]),
+        .init(name: "Swinject/Swinject", requirement: .exact("2.7.1"), platforms: [.macOS]),
```

Comment:
With the goal of minimizing the time it takes for the acceptance test to run, I'd reduce this list to only one framework. Unless the second one adds a new scenario, which I don't think it does.